Resources/config/services.yml

```diff
@@ -3,6 +3,7 @@ services:
       alias: dmishh.settings.settings_manager
 
     dmishh.settings.settings_manager:
+        public: true
```

Comment:
Why should this service be public?

Reply:
shouldn't it be public to prevent BC breaks ?

Reply:
Okey, I see. Thanks

Reply:
👍

Reply:
Not working in SF4.1, the alias need to be **public** like:
```yaml
settings_manager:
  alias: dmishh.settings.settings_manager
  public: true
```